notebooks/Query_butler_WEP_outputs.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Do you want to add an example showing what's inside a donut stamp object in addition to the image? Like just print one out showing that there is centroid, blending, and other info inside?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/aos_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>